Documentation/project-docs/platform-names.md

```diff
@@ -4,6 +4,26 @@ Over the years, we've accumulated quite a few names, platforms, and components t
 
 This will also list some aliases. As you'll see the aliases aren't always correct -- they are merely listed to help you find the better and less confusing terminology.
 
+In this document, the following terms are used:
+
+* **IL**. Intermediate language. IL is a stack-based instruction set and is what higher level languages, such as C#, compile down to.
+
+* **GC**. Garbage collector. Garbage collection is an implementation of automatic memory management. .NET currently uses a generational garbage collector, i.e. it groups objects into generations to limit the number of nodes it has to walk for determining which objects are alive. This speeds up collection times.
+
+* **JIT**. Just in time compiler. This technology compiles IL to machine code that the processor understands. It's called JIT because compilation happens on demand and is performed on the same machine the code needs to run on. Since JIT compilation happens during execution of the application, compile time is part of the run time. Thus, JIT compilers have to trade spending more time optimizing code with the savings the resulting code can produce.
```

Comment:
Try with "into" instead of "to" ...

> This technology compiles IL into machine code that the processor understands.

Refactor to remove the ending preposition ...

> It's called JIT because compilation happens on demand and is performed on the same machine the code needs to run on.

... to ...

> It's called JIT because compilation happens on demand on the target machine.

Side ? on this phrase ... `compile time is part of the runtime` ... is it better to say `compile time is part of the initial runtime`? ... could be my misunderstanding regarding how often the initial code is compiled. Does JIT re-compile every run? If not, I'd say `initial runtime` here. If it's every run, then no problem here..

This line is opaque ...

> Thus, JIT compilers have to trade spending more time optimizing code with the savings the resulting code can produce.

I can't understand it outside of perhaps some kind of trade-off exists with JIT compilers. I suggest elaborating a bit to clarify it.